subscriptions/vs-rstudio.md

```diff
@@ -100,27 +97,30 @@ RStudio is available in open source and commercial editions and runs on the desk
+
 A:  No.  The offer will expire on June 1, 2019. You must complete the entire activation process before the expiration date in order to use the benefit.
 
 ### Q: I don't want to sign up for a recurring subscription.  Can I opt out?
-A:  You'll be automatically signed up for the recurring subscription, but you can cancel any time before the renewal date and still take advantage of the first year of RStudio that's included with your Visual Studio subscription at no charge.  
 
-After you complete the signup for RStudio, you'll receive several emails, including one inviting you to manage your subscription for RStudio on BillingPortal.com.  Accepting the invitation will allow you to change plans, manage stored credit card information, and manage your account details. 
+A:  You'll be automatically signed up for the recurring subscription, but you can cancel any time before the renewal date and still take advantage of the first year of RStudio that's included with your Visual Studio subscription at no charge.
+
+After you complete the signup for RStudio, you'll receive several emails, including one inviting you to manage your subscription for RStudio on BillingPortal.com.  Accepting the invitation will allow you to change plans, manage stored credit card information, and manage your account details.
 
 ### Q:  If I don't have "R" installed, where can I get it?
-A:  If you receive an error that you don't have "R" installed, the error message will contain a link to the "Comprehensive R Archive Network (CRAN)" at https://cran.rstudio.com/.  You can also perform an internet search for other download locations. 
+
+A:  If you receive an error that you don't have "R" installed, the error message will contain a link to the "Comprehensive R Archive Network (CRAN)" at https://cran.rstudio.com/.  You can also perform an internet search for other download locations.
 
 ## Support resources
+
 -  RStudio self-help: https://support.rstudio.com/hc/
 -  RStudio Community: https://community.rstudio.com/
--  Assisted support:  https://support.rstudio.com/hc/requests/new 
+-  Assisted support:  https://support.rstudio.com/hc/requests/new
 -  For assistance with sales, subscriptions, accounts and billing for Visual Studio Subscriptions, contact Visual Studio [Subscriptions Support](https://visualstudio.microsoft.com/subscriptions/support/).
 -  Have a question about Visual Studio IDE, Azure DevOps Services or other Visual Studio products or services?  Visit [Visual Studio Support](https://visualstudio.microsoft.com/support/).
 
-
-
 ## Next steps
-The RStudio benefit is available for a very limited time.  Remember to claim your code and create your RStudio account before July 1, 2019.  
\ No newline at end of file
+
+The RStudio benefit is available for a very limited time.  Remember to claim your code and create your RStudio account before July 1, 2019.
```

Comment:
Maybe this should be at the top with an `[!IMPORTANT]` or `[!NOTE]`

Reply:
@nschonni - Thanks for the feedback.  The note is already there, however.

Reply:
Gotcha, I see it up on L19 now